src/base/utils.js

```diff
@@ -40,7 +40,7 @@ export function formatTime(time) {
       out += days + ":"
       if (hours < 1) out += "00:"
     }
-    if (hours && hours > 0) out += ("0" + hours).slice(-2) + ":"
+    if (hours && hours > 0 || paddedHours !== undefined && paddedHours) out += ("0" + hours).slice(-2) + ":"
```

Comment:
I wasn't sure if formatTime was used elsewhere, if so paddedHours wouldn't be a boolean.